torch/_inductor/cudagraph_trees.py

```diff
@@ -1292,7 +1337,7 @@ def add_function(
 
         # container needs to set clean up when fn dies
         get_container(self.device_index).add_strong_reference(fn)
-        return fn
+        return fn, fn(inputs)
```

Comment:
the type signature above is no longer right (are you running mypy on this file?)

Reply:
Did not realize I needed to enable mypy on a file.. I guess not

Reply:
Fixed. Will do full mypy enable in a follow up since.